pkg/loadbalancer/experimental/config.go

```diff
@@ -40,30 +41,51 @@ type TestConfig struct {
 	// NodePortAlg mirrors option.Config.NodePortAlg. This can be removed when the NodePort config
 	// flags move away from option.DaemonConfig and can thus be set directly.
 	NodePortAlg string `mapstructure:"node-port-algorithm"`
+
+	// EnableHealthCheckNodePort is defined here to allow script tests to enable this.
+	// Can be removed once this option moves out from DaemonConfig into [Config].
+	EnableHealthCheckNodePort bool `mapstructure:"enable-health-check-nodeport"`
 }
 
 func (def TestConfig) Flags(flags *pflag.FlagSet) {
 	flags.Float32("lb-test-fault-probability", def.TestFaultProbability, "Probability for fault injection in LBMaps")
 	flags.String("node-port-algorithm", option.NodePortAlgRandom, "NodePort algorithm")
+	flags.Bool("enable-health-check-nodeport", false, "Enable the NodePort health check server")
```

Comment:
mark hidden?

Reply:
The TestConfig is only used by tests so no need to hide it.